src/types/types.ts

```diff
@@ -439,3 +439,14 @@ export const enum ErrorStatusCode {
   /** @see https://docs.meilisearch.com/errors/#dump_not_found */
   DUMP_NOT_FOUND = 'dump_not_found',
 }
+
+export type TokenIndexRules = {
+  [field: string]: any // I have to check
```

Comment:
My bad, I forgot to remove it. 
I left the possibility to the user to add whatever field he wants here: 
```js
const searchRules = {
  "index": {
      "filter": "id = 1", 
      "customField": true
   }
}
```

So that he can add additional rules for his own back-end usages if they want to and avoid having to walk around with to many tokens as they could add everything they want in one. 

Which is possible in non typed languages